frontend/src/scenes/surveys/constants.tsx

```diff
@@ -314,6 +320,21 @@ export const defaultSurveyTemplates = [
     },
 ]
 
+export const errorTrackingSurvey: SurveyTemplate = {
```

Comment:
I also set 2 more settings on this survey so its a bit less annoying:
Wait period after seeing survey: 14 days, otherwise its gonna be super annoying if you have a bunch of exceptions
and 'Delay survey popup by at least'; 2 seconds so the UI has time to render a possible error message, the user has time to think of whats happened before changing the context right away
WDYT?

Reply:
Good call - added both!